dlp/snippets/deid.py

```diff
@@ -937,6 +937,80 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_simple_word_list]
+def deidentify_with_simple_word_list(
+    project,
+    input_str,
+    info_type,
+    word_list
+):
+    """Uses the Data Loss Prevention API to deidentify sensitive data in a
+      string by matching against custom word list.
+
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        input_str: The string to deidentify (will be treated as text).
+        info_type: A string representing info type to look for.
+            A full list of info type categories can be fetched from the API.
+        word_list: The list of strings to match against.
+
+    Returns:
+          None; the response from the API is printed to the terminal.
+    """
+    # Import the client library
+    import google.cloud.dlp
+
+    # Instantiate a client.
+    dlp = google.cloud.dlp_v2.DlpServiceClient()
+
+    # Convert the project id into a full resource id.
+    parent = f"projects/{project}"
```

Comment:
move down to where you are using it

Reply:
In other samples, "parent" is initialized at the start, though I have updated in the next push. Please review. Thank you!